### PR TITLE
[TEDEFO-1101]: Load SdkCodelist, SdkField and SdkNode implementations using annotations

### PR DESCRIPTION
**Pull request by meletev**
…sing annotations.

### CODE REVIEW

**Review by rousso — DISMISSED** [review]
Unit tests do not pass.

**Review by rouschr — APPROVED** [review]
There are many small changes due to refactoring and renaming nature of the work so it is difficult to have an overview.
I will accept this so that it can be merged.

File: src/main/java/eu/europa/ted/eforms/sdk/selector/resource/SdkUnpacker.java [hunk 1]
```diff
@@ -34,32 +34,28 @@ public static void unpack(File archive, Path targetDir) throws IOException {
...
+                RegExUtils.removeFirst(entry.getName(), "eforms-sdk/"));
+
+            try {
+              if (entry.isDirectory()) {
+                log.trace("Creating directory [{}]", targetEntryPath);
+                Files.createDirectories(targetEntryPath);
+              } else {
+                try (InputStream fileInput = file.getInputStream(entry);
+                    FileOutputStream fileOutput = new FileOutputStream(targetEntryPath.toFile());) {
+                  IOUtils.copy(fileInput, fileOutput);
```
Comment (rouschr) [inline src/main/java/eu/europa/ted/eforms/sdk/selector/resource/SdkUnpacker.java:49]: Looks good

**Review by rouschr — APPROVED** [review]
Looks good.

**Review by rousso — APPROVED** [review]

**Pull request merged**